### PR TITLE
Made database timezone configurable

### PR DESCRIPTION
**Pull request by lesniakania**
Pull request for #144

### CODE REVIEW

File: groupdate.gemspec [hunk 1]
```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |spec|
     spec.add_development_dependency "activerecord-jdbcmysql-adapter"
   else
     spec.add_development_dependency "pg"
+    spec.add_development_dependency "pry"
```
Comment (eamonn-webster) [inline groupdate.gemspec]: I would remove this

Comment (lesniakania) [inline groupdate.gemspec]: it's just a development dependency, but ok

**Comment by ankane** [comment]
Hey @lesniakania, thanks for the PR! 👍  This is a great feature. I think we'll need a good number of tests to add this. For PostgreSQL and Redshift, it looks like `:database_time_zone` isn't used in the query.

**Comment by ankane** [comment]
Cleaning up PRs. Feel free to address the above and reopen.

**Pull request closed without merging**